### ISSUE TITLE
View of Peer Components grade by TA/instructor user

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#10408]
**Background**
We have an assignment with both non-peer and peer components.

The usual usage will be that:
* a TA or instructor will grade the non-peer components (the teaching staff work together to fill out a shared grade for each component, they can change the score saved by another member of the teaching staff
* multiple peers will grade the peer components -- we will have separate scores from each peer that are averaged.  
 
Usually the TA & instructors do NOT save scores on the peer components. 

**Expected behavior**
So when a member of the teaching staff is looking at the rubric for a student with completed grading, it should look like this:  [ note the grey ungraded pill on the purple border peer component ]

<img width="153" alt="Screenshot 2024-04-29 at 11 59 00 AM" src="https://github.com/Submitty/Submitty/assets/5871417/e40846d0-8754-4679-8958-2dc0e77e5fbc">

**Bug**
However, right now the interface is incorrectly grabbing a score from one of the (non-teaching staff) peer graders for the peer components:  

<img width="134" alt="Screenshot 2024-04-29 at 11 58 32 AM" src="https://github.com/Submitty/Submitty/assets/5871417/6fdc1b8a-012b-4f6e-825d-4eeab3b0f12a">

In this example the purple border peer component has not been graded by any member of the teaching staff.  The note "Graded by XXXX" lists the name of a student in the course who was assigned and completed a peer score for this student.    We should not see peer scores entered by anyone other us in this panel.  The "Peer Grading" panel should display the scores by other users (students or other members of the teaching staff).

**Comment by Xiayucheng1212** [comment]
@bmcutler I would like to clarify the preferred behavior is that the peer component should show the scores completed by other peer graders and keeps gray until the teaching staff gives a score? Is that correct?

**Comment by micpap25** [comment]
@bmcutler bringing to your attention again since it seems @Xiayucheng1212 is interested

**Comment by bmcutler** [comment]
@Xiayucheng1212 
Yes, the peer component should be grey UNLESS it was graded by the current user.
And in typical usage the teaching staff does not fill out the peer components.

**Comment by micpap25** [comment]
Note that fixing this issue should also remove the ongoing issue that you can attempt to "Verify Grader" on a student (which is NOT intended).
